src/Tribe/Editor/Compatibility/Classic_Editor.php

```diff
@@ -236,7 +236,7 @@ public function filter_tribe_editor_should_load_blocks( bool $should_load_blocks
 		}
 
 		if ( static::get_classic_override() ) {
-			$should_load_blocks = true;
+			$should_load_blocks = ! $should_load_blocks;
```

Comment:
The previous version of the code would set `$should_load_blocks` to `true`. This one to the opposite of the previous value. The logic went from constant to contextual, is this correct? It might be, I'm asking as I would not know.

Reply:
Correct - the override "reverses" whatever is the current setup - so classic to block and block to classic.
ie: assuming we're overriding,  if `classic_editor` is set, then we get blocks, otherwise we get calssic